src/otx/algo/visual_prompting/zero_shot_segment_anything.py

```diff
@@ -162,11 +172,12 @@ def _point_selection(
         # sample fg_coords_scores matched by matched_matrix
         matched_grid = fg_coords_scores.unsqueeze(1) * matched_matrix.unsqueeze(-1)
 
-        # sample the highest score one of the samples that are in the same grid
-        points_scores = matched_grid[matched_grid[..., -1].argsort(dim=0, descending=True)[0]].diagonal().T
+        matched_indices = matched_grid[..., -1].topk(k=1, dim=0, largest=True)[1][0].to(torch.int64)
```

Comment:
minor comment: with `topk` operation output might be already integer.

Reply:
Thanks, it was updated for export phase, but it is unnecessary anymore as you said. I'll update it in the next commit or PR.